RedisChecker/redisChecker.go

```diff
@@ -35,18 +42,44 @@ func (m *RedisConnection) Validate() (bool, error) {
 func (m *RedisConnection) Check() (bool, bool, error) {
 	ctx := context.Background()
 
-	rdb := redis.NewClient(&redis.Options{
-		Addr:     m.BuildConnectionString(),
-		Password: m.Password, // no password set
-		DB:       m.Database, // use default DB
-	})
+	if !m.IsClustered {
```

Comment:
Let's change it in a way that we can put Sentinel in place for the future.
It can be a switch-case and three private func (for now two private func) which actually do the checking.